src/quacc/recipes/mlp/_base.py

```diff
@@ -56,10 +62,20 @@ def pick_calculator(
 
         calc = CHGNetCalculator(**kwargs)
 
+    elif method.lower() == "mace-mp-0":
+        from mace import __version__
+        from mace.calculators import mace_mp
+
+        if "default_dtype" not in kwargs:
+            kwargs["default_dtype"] = "float64"
+        calc = mace_mp(**kwargs)
+
     elif method.lower() == "mace":
         from mace import __version__
         from mace.calculators import mace_mp
 
+        mace_warning()
+
         if "default_dtype" not in kwargs:
             kwargs["default_dtype"] = "float64"
```

Comment:
You can condense this to

```python
elif method.lower() in ["mace", "mace-mp-0"]:
    from mace import __version__
    from mace.calculators import mace_mp


    if method.lower() == "mace":
        mace_warning()

    ...
```
to reduce code duplication.

Reply:
Not sure what I was thinking there, sounds good!